stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -221,10 +221,23 @@ public void onChildInserted(Object parentElement, Object previousElement, Object
 
     @Override
     public void onChildRemoved(Object parentElement, Object childElement) {
-      ChildNodeRemovedEvent message = new ChildNodeRemovedEvent();
-      message.parentNodeId = mObjectIdMapper.getIdForObject(parentElement);
-      message.nodeId = mObjectIdMapper.getIdForObject(childElement);
-      mPeerManager.sendNotificationToPeers("DOM.childNodeRemoved", message);
+      Integer parentNodeId = mObjectIdMapper.getIdForObject(parentElement);
+      Integer childNodeId = mObjectIdMapper.getIdForObject(childElement);
+
+      if (parentNodeId == null || childNodeId == null) {
+        LogUtil.w(
+            "DOM.onChildRemoved() called for a non-mapped node: "
```

Comment:
Does this spam in practice?  I want to avoid confusing folks with internal error messages.

Specifically, if this is expected and/or fires often, consider using debug or info level.